scripts/plotting/global_latlon_vect_map.py

```diff
@@ -449,5 +500,19 @@ def global_latlon_vect_map(adfobj):
     #Notify user that script has ended:
     print("  ...lat/lon vector maps have been generated successfully.")
 
+#########
+# Helpers
+#########
+
+def _get_climo_yrs(cam_ts_loc):
+    starting_location = Path(cam_ts_loc)
+    files_list = sorted(starting_location.glob('*nc'))
+    try:
+        syear = int(files_list[0].stem[-13:-9])
+        eyear = int(files_list[0].stem[-6:-2])
+    except:
+        print("Smoethign is borken...")
+    return syear, eyear
+
```

Comment:
If we save the start and end years as part of the ADF object in `adf_info.py` then this function and related code isn't needed.